components/src/Checkbox/Checkbox.js

```diff
@@ -64,13 +64,15 @@ const VisualCheckbox = styled.div`
   }
 `;
 
-const CheckboxWrapper = styled.label`
+const CheckboxClickable = styled.label`
```

Comment:
What do you think about extracting a `ClickableArea` component?

I ask because I noticed that `CheckboxClickable`, `RadioClickable`, and `ToggleClickable` are identical.

Reply:
Or...`InputClickableArea`?
I realized that it's a label, to select the form input.